_inc/client/plans/single-product-backup.jsx

```diff
@@ -51,6 +58,80 @@ export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
 	);
 }
 
+class SingleProductBackupBody extends React.Component {
+	static propTypes = {
+		backupPlanPrices: PropTypes.object,
+		currencySymbol: PropTypes.string,
+		setSelectedBackupType: PropTypes.func,
+		selectedBackupType: PropTypes.string,
+		upgradeLinks: PropTypes.object,
+	};
+
+	handleSelectedBackupTypeChange = event => {
+		this.props.setSelectedBackupType( event.target.value );
+	};
+
+	render() {
+		const { currencySymbol, backupPlanPrices, selectedBackupType, upgradeLinks } = this.props;
+
+		const upgradeTitles = {
+			'real-time': __( 'Upgrade to Real-Time Backups' ),
+			daily: __( 'Upgrade to Daily Backups' ),
+		};
+
+		const backupOptions = [
+			{
+				type: 'daily',
+				name: __( 'Daily Backups' ),
+				price: backupPlanPrices.jetpack_backup_daily.yearly,
+			},
+			{
+				type: 'real-time',
+				name: __( 'Real-Time Backups' ),
+				price: backupPlanPrices.jetpack_backup_realtime.yearly,
+			},
+		];
+
+		return (
+			<React.Fragment>
+				<p>
+					{ __(
+						'Always-on backups ensure you never lose your site. Choose from real-time or daily backups. {{a}}Which one do I need?{{/a}}',
+						{
+							components: {
+								a: <ExternalLink href="https://jetpack.com/upgrade/backup/" icon iconSize={ 12 } />,
```

Comment:
It seems that just using `ExternalLink` is not enough. You'll need to add `target="_blank"` for it to really open in a new window.